oaknational/Oak-Web-Application
# fix(video-tracking): track end at 90 percent

## Description

- call the tracking `Video Finished` event at 90 percent

## Issue(s)

Fixes #1067 

## How to test

1. Go to {cloud link}
2. Play video past 90%
3. Should get a video tracking event in posthog
4. Should only get a single event page page view



## Checklist
